chore: durability updates for TSM snapshots (#3944)
influxdata/docs-v2: content/influxdb/cloud/reference/internals/durability.md

[TSM](/influxdb/cloud/reference/glossary/#tsm-time-structured-merge-tree) files
in object storage.

### Periodic TSM snapshots
To provide multiple data recovery points, InfluxDB Cloud takes weekly snapshots of TSM files uploaded to object storage. The TSM snapshot includes a copy of all (non-deleted) data when the snapshot is created.
These snapshots are preserved for 100 days.

## Recovery
InfluxDB Cloud uses the following out-of-band backups stored in object storage to recover data:

- **Message queue backup:** line protocol from inbound write requests within the last 96 hours
- **Compaction backup:** TSM files

> - **TSM Snapshots:** Weekly snapshots of TSM files in objectstore

Reviewer: ```suggestion
- **TSM snapshots:** Weekly snapshots of TSM files in objectstore
```

[TSM](/influxdb/cloud/reference/glossary/#tsm-time-structured-merge-tree) files
in object storage.

### Periodic TSM snapshots

> Periodically the TSM files uploaded to object storage are snapshoted.
> Each TSM snapshot is a complete copy of all non-deleted data, as of the time the TSM snapshot is created.
> These snapshots are created weekly and preserved for 100 days, providing multiple recovery points.

Reviewer: ```suggestion
To provide multiple data recovery points, InfluxDB Cloud takes weekly snapshots of TSM files uploaded to object storage. The TSM snapshot includes a copy of all (non-deleted) data when the snapshot is created.
These snapshots are preserved for 100 days.
```

## Recovery
InfluxDB Cloud uses the following out-of-band backups stored in object storage to recover data:

- **Message queue backup:** line protocol from inbound write requests within the last 96 hours
- **Compaction backup:** TSM files
- **TSM Snapshots:** Weekly snapshots of TSM files in objectstore

The Recovery Point Objective (RPO) is any accepted write.
The Recovery Time Objective (RTO) is harder to definitively predict as potential failure modes can vary.
While most common failure modes can be resolved within minutes or hours,
critical failure modes may take longer.

> For example, if we need to rebuild all data from the TSM Snapshots and message queue backup,

Reviewer: ```suggestion
For example, if we need to rebuild all data from the TSM snapshots and message queue backup,
```
